src/views/integrations/tickets-wallet-plus/pdf/pass/body/date.php

```diff
@@ -0,0 +1,34 @@
+<?php
+/**
+ * PDF Pass: Body - Event Time
```

Comment:
In favor of consistency, we should call it event date. We're using event date for the pieces that include the event date. When we use time, it refers only to the time piece.